src/libs/OptionsListUtils.js

```diff
@@ -297,13 +297,13 @@ function getOptions(reports, personalDetails, draftComments, activeReportID, {
             return;
         }
 
-        const hasDraftComment = report
+        const reportDraftComment = report
```

Comment:
Similar comment here. I don't get why this variable change is relevant to your PR.

Reply:
Just to make more sense of it. As the end behaviour is changed. Previously, the variable was telling us that do we have a draft comment. Now it only store the draftComment object and we check for the Presence of draft message in later logic. So I thought change the variable name to reflect it.